index.js

```diff
@@ -1,11 +1,14 @@
+import { Blob } from 'node:buffer';
```

Comment:
This is not compatible with the targeted Node.js version.

Reply:
Did you mean that nodejs v12 does not support Blob ?

Reply:
Yes. But instead of asking, you could have easily checked this yourself. Simply running the tests would have shown you this.